### PR TITLE
Add python3.md

### PR DESCRIPTION
**Pull request by rselwyn**
I pretty much took the python.md and added a similar python3.md.  Just `python` will call up a python 2 interpreter, but `python3` will call a python 3 interpreter.  Because they are different, I feel having a different page would be appropriate.

### CODE REVIEW

**Comment by rselwyn** [comment]
@rprieto @waldyrious @felixonmars Can this be merged?

**Comment by igorshubovych** [comment]
Sure. Please give us some time to review. We are currently overwhelmed by upcoming requests.

**Comment by rselwyn** [comment]
No problem.

**Comment by igorshubovych** [comment]
Interesting.
I have brew installed Python 3 on my OSX machine with python3 (!) alias, not just python.

**Comment by rselwyn** [comment]
On my computer (OS X), I have both Python 2 and 3 installed.  Because of this, `python` launches python 2 and `python3` launches python3.

File: pages/common/python3.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Python3
+
+> Python 3 language interpeter
+
+- Call a Python 3 interactive shell
+
+`python`
```
Comment (Kwpolska) [inline pages/common/python3.md]: Shouldn’t this be `python3`?

**Comment by ostera** [comment]
I feel this is unnecessary unless it clarifies differences only available for python3

**Comment by waldyrious** [comment]
I agree with @ostera

**Comment by rselwyn** [comment]
Ok - I was wondering if this edit was too specific.

**Pull request closed without merging**